foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -30,8 +30,16 @@ Finally, this is your first JavaScript program built from scratch, so don't forg
    console.log(playRound(playerSelection, computerSelection));
    ~~~
 
-6. Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-   1. You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
+6. Write a NEW function called `game()`. Call the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
+   1. Remember loops? This is a great opportunity to use one to play those five rounds:
```

Comment:
Did we actually introduce loops by now? Also, can we start at 0 not 1?

Reply:
[Problem solving](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/problem-solving) lesson covers FizzBuzz and explains everything step-by-step, loop included. It is true that loops are covered in [Fundamentals Part 4](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/fundamentals-part-4) that come after the Project - after `Clean Code` and `Installing Node.js` lessons to be precise - so the users are expected to learn loops without picking up new concepts besides Arrays afterwards. They're ready and the loop they've seen is enough for their needs - they're supposed to call their `playRound()` function five times so it's only matter of putting it there. The only thing left to figure out is where to put win variables and how to interact with them.


We can start at 0 not 1, I'll just make a small case for this to make sure we're not losing some potential upside.
Problem Solving lesson also uses start at 1 and gives this reasoning while explaining how the loop works:
>If you haven’t seen this before and it looks strange, it’s actually straightforward. We declare a variable i and assign it 1: the initial value of the variable i in our loop (NOTE: Most of the time, programmers find themselves looping from 0. Due to the needs of our program, we’re starting from 1). The second clause, i <= answer is our condition. We want to loop until i is greater than answer. The third clause, i++, tells our loop to increment i by 1 every iteration. As a result, if the user inputs 10, this loop would print numbers 1 - 10 to the console.

Obviously, there's no *need* for it to start at 1 here but it might feel more clear thanks to familiarity and it might feel "more right" to them, since it goes 1-5 as if it were Round 1, Round 2 and so on. Might prevent some unwanted tinkering with how the loop is constructed.

Reply:
> Obviously, there's no need for it to start at 1 here
Since we've established both that and convention tends to be starting at 0, I think we should go with 0. I also don't want people to default to starting at 1.

Reply:
Some things are just as much about the ability to follow the instructions (AKA, not "tinkering"), you know?

Reply:
Makes absolute sense all you've written. Going to change it so it starts at 0. I assume that using a loop is a go, correct?

Reply:
Yes, let's please use a loop.